### PR TITLE
El capitan

### PR DESCRIPTION
**Pull request by JanX2**
The new graphics are by @robotspacer. 

I don’t require support for older systems. If you require help in merging the updated widget graphics so that older systems still display the correct ones, feel free to contact me. :)

I have also unified the indentation to use spaces throughout, updated the README and disabled ALWAYS_SEARCH_USER_PATHS as recommended by Xcode.

### CODE REVIEW

**Comment by Kentzo** [comment]
Hi @JanX2 

That's quite a work!
I'll be looking at it during the week (mostly on the weekend).

> I don’t require support for older systems.

I was thinking about moving to 3.x and deprecating support for older system (except for bug fixes).
Is 10.12 the min supported version for these changes?

**Comment by robotspacer** [comment]
My changes support 10.11 as well. I just did some quick tests with the demo app and it looks like 10.10 works fine too. 10.7 through 10.9 all have significant issues.

**Comment by Kentzo** [comment]
Do you mind if I will rebase / squash commits when I'll try to maintain compatibility?

**Comment by robotspacer** [comment]
Not at all, you're welcome to do what you like with my changes.

**Comment by Kentzo** [comment]
Things I would like to integrate:

- xcassets
- smaller text when there is no shortcut

Did you fix any errors along the way?

**Comment by robotspacer** [comment]
My version includes different images designed to better fit the appearance of macOS 10.11 and later, especially on Retina displays. There are a lot of changes in 654f8576eb075ae1d80cfc59bb8b44d6bb1f3095 and 409d03fbb20bc69c8c21cb0b4921a5832650fe6e there are necessary to draw the new graphics correctly. So if you wanted to include the new graphics, you would need to add a lot of version checks to make sure everything is correct in each OS version. That's up to you of course!

**Comment by Kentzo** [comment]
Sure, I'll take another round sometime this week. Thank you again for the work.

**Comment by JanX2** [comment]
@Kentzo 
Are there any issues with this that need to be resolved? Apart from the merge conflict that is. :)

**Comment by Kentzo** [comment]
@JanX2 Not sure yet. Will be looking into this soon as I'm doing an uplift of an app that uses Shortcut Recorder.

File: Library/SRValidator.m [hunk 1]
```diff
@@ -211,23 +215,20 @@ - (NSString *)SR_path
...
-    if ([path length] > 1)
-        [path deleteCharactersInRange:NSMakeRange([path length] - 1, 1)];
+	NSMutableArray *titles = [[NSMutableArray alloc] initWithCapacity:items.count];
+    for (NSMenuItem *menuItem in items) {
+		NSString *title = menuItem.title;
+		if (title) [titles addObject:title];
+	}
 
-    return path;
+    return [titles componentsJoinedByString:@" → "];
```
Comment (Kentzo) [inline Library/SRValidator.m]: I believe I've seen system UI to use "a➝b", not "a → b". But cannot recall where.

File: .gitignore [hunk 2]
```diff
@@ -16,3 +16,4 @@ profile
 
 ## Ignore incredibly annoying .DS_Store files
 .DS_Store
+Products/*
```
Comment (Kentzo) [inline .gitignore]: Doesn't Xcode place Products outside of the project's directory?

**Comment by Kentzo** [comment]
Ok, so plan is to get dissect this PR into multiple parts. New UI will be part of Shortcut Recorder 3. Everything else that is still relevant for Shortcut Recorder 2 will be part of the last release of the version 2.

New base SDK for SR 3 will be either 10.10 or 10.11

File: ShortcutRecorder.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -309,6 +309,7 @@
 			);
 			name = ShortcutRecorder;
 			sourceTree = "<group>";
+			usesTabs = 0;
```
Comment (Kentzo) [inline ShortcutRecorder.xcodeproj/project.pbxproj]: Implemented by adaf58c

**Comment by Kentzo** [comment]
@JanX2 Is there a difference between PSD and Sketch?

**Pull request closed without merging**